tests/structurator/insert_wu_test.py

```diff
@@ -0,0 +1,55 @@
+from uuid import uuid4
+
+from _pytest.fixtures import fixture
+
+from sampo.schemas.graph import EdgeType
+from sampo.schemas.works import WorkUnit
+from sampo.structurator.insert_wu import insert_work_unit
+
+
+@fixture()
+def setup_wu() -> WorkUnit:
+    return WorkUnit(str(uuid4()), 'test_wu')
+
+
+# TODO docstring documentation ??
```

Comment:
Remove TODO. Tests are not the part of API, so documentation is redundant.

Reply:
Tests should be written in paradigm `You don't need the documentation`.
Them should be clear without it.